test/e2e/scenarios/Cluster/cluster_update_with_tags.yaml

```diff
@@ -53,12 +53,12 @@ steps:
             value: "test_value_1"
           - key: "test_key_2"
             value: "test_value_2"
-#    expect_aws:
-#      Tags:
-#        - Key: "test_key_1"
-#          Value: "test_value_1"
-#        - Key: "test_key_2"
-#          Value: "test_value_2"
+    expect_aws:
+      Tags:
+        - Key: "test_key_1"
+          Value: "test_value_1"
+        - Key: "test_key_2"
+          Value: "test_value_2"
```

Comment:
How is this asserting that a cluster is created?

Reply:
This is only asserting tags field of a cluster. Cluster creation is validated (assert) at step 1 in this yaml test file.
```
    expect_k8s:
      spec:
        name: cluster$RANDOM_SUFFIX
    expect_aws:
      Name: cluster$RANDOM_SUFFIX